src/gui/attributetable/qgsdualview.cpp

```diff
@@ -979,6 +981,33 @@ void QgsDualView::showViewHeaderMenu( QPoint point )
   connect( sort, &QAction::triggered, this, [ = ]() {modifySort();} );
   mHorizontalHeaderMenu->addAction( sort );
 
+  mHorizontalHeaderMenu->addSeparator();
+
+  QAction *fieldCalculator = new QAction( tr( "&Field Calculator..." ), mHorizontalHeaderMenu );
```

Comment:
```suggestion
  QAction *fieldCalculator = new QAction( tr( "Open &Field Calculator..." ), mHorizontalHeaderMenu );
```
consistent with other actions label

Reply:
Yep, I can't be against consistency!